.kokoro/generate-docs.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+# Disable buffering, so that the logs stream through.
+export PYTHONUNBUFFERED=1
+
+export PATH="${HOME}/.local/bin:${PATH}"
+
+# Install dependencies.
+python3 -m pip install --user --quiet --upgrade nox
+python3 -m pip install --user gcp-docuploader
+
+# Retrieve repositories to regenerate the YAML with.
+for bucket_item in $(gsutil ls gs://docs-staging-v2 | grep "docfx-python"); do
+
+  # Parse the library name.
+  library=$(echo ${bucket_item} | cut -d "-" -f 5)
+
+  # Only process unique entries.
```

Comment:
Can use `sort` to only process unique library entries. Something like this:

```
LIBRARIES=$(gsutil ls 'gs://docs-staging-v2/docfx-python*' | sort -u -t- -k5,5)
```

Reply:
This is much better, thank you!